backend/eventposts/views.py

```diff
@@ -29,10 +29,12 @@ class EventViewSet(viewsets.ModelViewSet):
     lookup_field = "id"
 
     def wrap(self, request, data):
+        queryset = User.objects
+        owner = queryset.filter(id=data["owner"]).get().username
         response = \
-            {"@context": "https://www.w3.org/ns/activitystreams", "summary": str(request.user) + " created an event",
+            {"@context": "https://www.w3.org/ns/activitystreams", "summary": owner + " created an event",
```

Comment:
do these changes create any problem with already existing eventposts tests?

Reply:
They should not, in fact this should fix a bug where the response showed the requesting user as the owner of any post, instead of the actual owner. Still, I will do a few more manual tests to make sure it's fine.

Reply:
I have carried out the tests, everything seems fine.